tests/components/upnp/test_config_flow.py

```diff
@@ -203,5 +128,33 @@ async def test_flow_config(hass: HomeAssistantType):
         assert result["data"] == {
             CONFIG_ENTRY_ST: mock_device.device_type,
             CONFIG_ENTRY_UDN: mock_device.udn,
-            CONFIG_ENTRY_SCAN_INTERVAL: DEFAULT_SCAN_INTERVAL,
         }
+
+
+async def test_options_flow(hass: HomeAssistantType):
+    """Test options flow."""
+    udn = "uuid:device_1"
+    mock_device = MockDevice(udn)
+    st = mock_device.device_type  # pylint: disable=invalid-name
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={CONFIG_ENTRY_UDN: udn, CONFIG_ENTRY_ST: st},
+        options={CONFIG_ENTRY_SCAN_INTERVAL: DEFAULT_SCAN_INTERVAL},
+    )
+    config_entry.add_to_hass(hass)
+
+    hass.data[DOMAIN] = {
```

Comment:
Can we set up the integration instead so we don't need to mock integration code?

Reply:
Changed.